Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -63,11 +63,30 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Rule names to fetch alerts by, separated by semicolon. If empty, all alerts
+    will be fetched
+  name: rule_names
+  required: false
+  type: 12
+- defaultvalue: 24 hours
+  display: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months,
+    1 year)
+  name: triggered
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
```

Comment:
I think you should add the `max_fetch` parameter so customers can set the max results to fetch.
WDYT?